man/systemd.resource-control.xml

```diff
@@ -691,6 +691,19 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AllowBindPorts=<replaceable>IP_PORT</replaceable></varname></term>
+
+        <listitem>
+          <para>Allow binding socket address only to certain ip ports specified by <replaceable>IP_PORT</replaceable>.
```

Comment:
Please add how multiple ports can be specified (e.g. space separated, comma separated, specify the property multiple times, etc.)

Reply:
@anitazha 
Great question.
Wdyt about port ranges `AllowBindPorts=2000:3000,` so no need to repetitively specify  `AllowBindPorts=` for adjacent values?

Reply:
Port range sounds good. I think properties that take ranges more commonly use a dash/hyphen (i.e. `SourcePort=` property can accept a port range separated by dash).

Reply:
@anitazha 

I've implemented port range support in https://github.com/systemd/systemd/commit/9647e774162847f07bcdc4b6709567a008318f48
It adds complexity to PR, so I won't add it until the initial hight level comments from maintainers.